app/views/login.scala.html

```diff
@@ -102,72 +101,40 @@ <h2 id="error-summary-heading">@Messages("label.checkTheForm")</h2>
-                
-                        <!-- Reference number -->
-                    <fieldset class="form-group">
-                        <legend class="form-label-bold">@Messages("label.referenceNumber") <span class="visuallyhidden" aria-hidden="true" >(Required)</span></legend>
-                        @errorWrapper(Seq(theForm("ref1"),theForm("ref2")),"form-reference"){
-                            <div class="form-grouped">
-
-                                @inputText(
-                                    field = theForm("ref1"),
-                                    '_label -> Messages("label.referenceNumber1"),
-                                    '_labelClass -> "visuallyhidden",
-                                    '_fieldsetClass -> "form-group-reference-7",
-                                    'class -> "form-control",
-                                    'maxlength -> "8",
-                                    'pattern -> "[0-9]*",
-                                    '_showConstraints -> false, 
-                                    '_showErrors -> false
-                                )
-
-                                <span class="form-group divider">/</span>
-
-                                @inputText(
-                                    field = theForm("ref2"),
-                                    '_label -> Messages("label.referenceNumber2"),
-                                    '_labelClass -> "visuallyhidden",
-                                    '_fieldsetClass -> "form-group-reference-3",
-                                    'class -> "form-control",
-                                    'maxlength -> "3",
-                                    'pattern -> "[0-9]*",
-                                    '_showConstraints -> false, 
-                                    '_showErrors -> false
-                                )
-                            
-                            </div>
-                        }
-                    </fieldset>
+
+                    @inputText(
+                        field = theForm("referenceNumber"),
+                        '_label -> Messages("label.referenceNumber"),
+                        '_labelClass -> "form-label",
+                        'class -> "form-control form-group-reference-7"
+                    )
 
                     <!-- Postcode -->
                     @inputText(
-                        field = theForm("postcode"),
-                        '_label -> Messages("label.postcode"),
-                        '_labelClass -> "form-label-bold",
-                        'class -> "form-control postcode",
-                        'maxlength -> "8",
-                        '_type -> "hidden"
+                    field = theForm("postcode"),
+                    '_label -> Messages("label.postcode"),
+                    '_labelClass -> "form-label",
+                    'class -> "form-control postcode",
+                    '_type -> "hidden"
```

Comment:
Should the text input have a maxlength and pattern?

Reply:
Same is true for the reference number above.

Reply:
Stephen. Dan wanted it removed as HTML5 validation is not recommended by gov design. I also added novalidation to form for same reason. Terry and Dan asked to remove max length too.
Do the scala tests look ok?

Reply:
Ok. That fine if Dan and Terry recommend it I will merge.